src/webhooks/registry.ts

```diff
@@ -78,8 +79,40 @@ function isSuccess(result: any, deliveryMethod: DeliveryMethod, webhookId?: stri
   }
 }
 
+function versionSupportsEndpointField(
```

Comment:
I wonder if we should just break `version` apart and check if `year > 2020 || month >= 07`, so we don't have to remember to add future versions here (even though the old check will be deprecated fairly soon).

Reply:
I've added a new utility method (named `versionCompatible`  for now) as you suggested. I wasn't sure how we wanted to handle unstable/unversioned so those always return `true`.